packages/react-components/src/components/AutoComplete/index.ts

```diff
@@ -0,0 +1,5 @@
+import { AutoComplete } from './AutoComplete';
+import { AutoCompleteProps } from './types';
+
+export { AutoComplete };
+export type { AutoCompleteProps };
```

Comment:
You need to add export statements to the `src/index.ts` file to make the AutoComplete component available for usage.